napari/layers/labels/_labels_key_bindings.py

```diff
@@ -90,6 +90,11 @@ def increase_label_id(layer: Labels):
     layer.selected_label += 1
 
 
+@register_label_mode_action(trans._("Toggle preserve labels"))
```

Comment:
this is wrong. `register_label_mode_action` is for restore `mode` of layer. There is a need to use a more generic decorator. 
```suggestion
@register_layer_attr_action(trans._("Toggle preserve labels"), "preserve_labels")
```

Reply:
I'm not able to get that to work?
`@register_label_action(trans._("Toggle preserve labels"),)`
does work though, is that OK—it's what the others mostly use, like 
```
@register_label_action(
    trans._("Increase the currently selected label by one."),
)
def increase_label_id(layer: Labels):
    layer.selected_label += 1
```